src/main/java/org/fundacionjala/coding/raul/movies/Customer.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.coding.raul.movies;
+
+import java.util.LinkedList;
+
+class Customer {
+    private String name;
+    private LinkedList<Rental> rentals = new LinkedList();
```

Comment:
This should be instantiate in the constructor.

Reply:
Done